icicle/include/icicle/fields/host_math.h

```diff
@@ -293,6 +454,21 @@ namespace host_math {
       }
     }
   }
+  template <unsigned NLIMBS>
+  static constexpr void get_higher_with_slack(const storage<2 * NLIMBS>& xs, storage<NLIMBS>& out, unsigned slack_bits)
+  {
+    for (unsigned i = 0; i < NLIMBS; i++) {
+      out.limbs[i] = (xs.limbs[i + NLIMBS] << 2 * slack_bits) + (xs.limbs[i + NLIMBS - 1] >> (32 - 2 * slack_bits));
+    }
+  }
+
+  template <unsigned NLIMBS>
+  static constexpr bool is_equal(const storage<NLIMBS>& xs, const storage<NLIMBS>& ys)
+  {
+    for (unsigned i = 0; i < NLIMBS; i++)
+      if (xs.limbs[i] != ys.limbs[i]) return false;
```

Comment:
what is wrong with memcmp?

Reply:
I did not change the logic of this code only the function name, I can change to memcpy if it's better